crates/sui-framework/sources/object.move

```diff
@@ -127,6 +127,12 @@ module sui::object {
     /// restrictable in the object's module.
     native fun borrow_uid<T: key>(obj: &T): &UID;
 
+    // === one-time witness ===
+
+    /// Tests if the argument type is a one-time witness, that is a type with only one instantiation
+    /// across the entire code base.
+    public native fun is_one_time_witness<T: drop>(_: &T): bool;
+
```

Comment:
A strange place for this. But not sure whereelse to put this... 

I see three options
- `sui::object`
- `sui::tx_context`
- A new module, `sui::types` or something similar

Reply:
I think `sui::types` is a good home.